scripts/build.ps1

```diff
@@ -56,6 +59,9 @@ param (
     [ValidateSet("x86", "x64", "arm", "arm64")]
     [string]$Arch = "x64",
 
+    [Parameter(Mandatory = $false)]
+    [switch]$Uwp = $false,
```

Comment:
Let's keep it all caps. `$UWP` like the `$PGO` option below.

Reply:
Done